egs/avspeech/local/loader/audio_feature_generator.py

```diff
@@ -0,0 +1,82 @@
+'''
+    This file deals with raw audio and converts them into
+    spectrogram.
+'''
+import librosa
+import numpy as np
+
+
+def convert_to_spectrogram(audio_raw: np.ndarray, sr=16_000, hann_length=400, hop_length=160, n_fft=512, p=0.3) -> np.ndarray:
```

Comment:
Also this is passed to the dataset, is this to support different types of features in an easier way?

Reply:
No, both these files were in the same directory previously. Since, the AVSpeechDataset moved into a different directory, this was a temporary fix. Since we will be using STFT from asteroid itself, I guess keep it as it is for now? As it is going to change anyway.

Reply:
I would change now to asteroid STFT actually ^^ Is it possible?